tests/x11/x3270_ssl.pm

```diff
@@ -70,7 +70,14 @@ sub run {
     # Launch x3270
     # Add noverifycert option if x3270 is or greater than v3.6ga
     my $noverifycert = ($current_ver < 3.6 ? '' : '-noverifycert');
-    script_run "x3270 -trace $noverifycert -tracefile $tracelog_file L:localhost:8443";
+
+    # Export SYSTEMD_PAGER to let the journalctl exits automatically
+    assert_script_run('export SYSTEMD_PAGER=""');
+
+    # Run x3270 as background since backend code adjust warning policy
+    # It introduces run error if the command is not exit
+    background_script_run("x3270 -trace $noverifycert -tracefile $tracelog_file L:localhost:8443", timeout => 90);
```

Comment:
I suggest you do some tire down operation after test completes

Reply:
yes. good comment, let me do it. thanks.

Reply:
Done.